tools/monitor.sh

```diff
@@ -0,0 +1,40 @@
+# path to the /dev/ serial device that the Teensy is connected to
+tty_path=""
+
+# Completely disable ctrl-C since it breaks the monitor
+trap '' INT;
```

Comment:
don't we want it to break when you ctrl-C? This seems like we should avoid ever doing this.

Reply:
If it's causing some horrible behavior we definitely don't want, I think there could be a way to fix that over just saying "no canceling allowed"

Reply:
also have to make sure we're not leaving a zombie monitor process by doing ctrl D instead.

Reply:
You said you verified that there's no zombie processes, so that's good.

Still want to check this myself, and see if I can make ctrl-c work properly as soon as I get the chance. If this is the only thing holding up the approval let me know.